src/main/java/runner/AppRunner.java

```diff
@@ -0,0 +1,79 @@
+package runner;
+
+import client.AllInformationGenerator;
+import client.ClientLogin;
+import java.util.Scanner;
+import movie.MoviesByFactors;
+
+public class AppRunner {
+
+  static final int ALL_CINEMAS = 1;
+  static final int MORE_INFORMATION = 2;
+  static final int ALL_MOVIES = 3;
+  static final int MOVIES_BY_NAME = 4;
+  static final int MOVIES_BY_FACTORS = 5;
+  static final int BUY_TICKETS = 6;
+  static final int SEE_ALL_INFORMATION = 7;
+  static final int EXIT = 8;
+
+  public static void run() {
```

Comment:
Usually, this method name is using in Classes that implement Runnable.

Reply:
fixed